rocketpy/Function.py

```diff
@@ -2270,15 +2322,326 @@ def integral(self, a, b, numerical=False):
+            inputs = self.__inputs__[:]
+            outputs = f"d({self.__outputs__[0]})/d({inputs[0]})"
+
+        # Create new Function object
+        return Function(source, inputs, outputs, self.__interpolation__)
+
+    def integralFunction(self, lower=None, upper=None, datapoints=100):
+        """Returns a Function object representing the integral of the Function object.
+
+        Parameters
+        ----------
+        lower : scalar, optional
+            The lower limit of the interval in which the function is to be
+            evaluated at. If the Function is given by a dataset, the default
+            value is the start of the dataset.
+        upper : scalar, optional
+            The upper limit of the interval in which the function is to be
+            evaluated at. If the Function is given by a dataset, the default
+            value is the end of the dataset.
+        datapoints : int, optional
+            The number of points in which the integral will be evaluated for
+            plotting it, which draws lines between each evaluated point.
+            The default value is 100.
+
+        Returns
+        -------
+        result : Function
+            The integral of the Function object.
+        """
+        if isinstance(self.source, np.ndarray):
+            lower = self.source[0, 0] if lower is None else lower
+            upper = self.source[-1, 0] if upper is None else upper
+            xData = np.linspace(lower, upper, datapoints)
+            yData = np.zeros(datapoints)
+            for i in range(datapoints):
+                yData[i] = self.integral(lower, xData[i])
+            return Function(
+                np.column_stack((xData, yData)),
+                inputs=self.__inputs__,
+                outputs=[o + " Integral" for o in self.__outputs__],
+            )
+        else:
+            lower = 0 if lower is None else lower
+            return Function(
+                lambda x: self.integral(lower, x),
+                inputs=self.__inputs__,
+                outputs=[o + " Integral" for o in self.__outputs__],
+            )
+
+    def inverseFunction(self, approxFunc=None, tol=1e-4):
+        """
+        Returns the inverse of the Function. The inverse function of F is a
+        function that undoes the operation of F. The inverse of F exists if
+        and only if F is bijective. Makes the domain the range and the range
+        the domain.
+
+        Parameters
+        ----------
+        lower : float
+            Lower limit of the new domain. Only required if the Function's
+            source is a callable instead of a list of points.
+        upper : float
+            Upper limit of the new domain. Only required if the Function's
+            source is a callable instead of a list of points.
+
+        Returns
+        -------
+        result : Function
+            A Function whose domain and range have been inverted.
+        """
```

Comment:
Docs here are wrong

Reply:
Should be fixed by d5c4eab967f2fab6416b9e327439f05d13482701.